test/fixtures/users.yml

```diff
@@ -4,4 +4,9 @@ user_1:
 
 user_2:
   email: test2@test.com
-  confirmed_at: <%= Time.now %>
\ No newline at end of file
+  confirmed_at: <%= Time.now %>
+
+admin_user:
+  email: admin@test.com
+  confirmed_at: <%= Time.now %>
+  admin: true
```

Comment:
Would it be better to use test@test.com as the admin user for consistency with the seeds?

Reply:
changed seeds so admin user is admn@test.com